services-geojson/src/main/java/com/mapbox/geojson/Polygon.java

```diff
@@ -375,21 +375,18 @@ public static TypeAdapter<Polygon> typeAdapter(Gson gson) {
    * ring rules.
    *
    * @param lineString {@link LineString} the polygon geometry
-   * @return true if number of coordinates are 4 or more, and first and last coordinates
-   *   are identical, else false
-   * @throws GeoJsonException if number of coordinates are less than 4,
+s   * @throws GeoJsonException if number of coordinates are less than 4,
    *   or first and last coordinates are not identical
    * @since 3.0.0
    */
-  private static boolean isLinearRing(LineString lineString) {
+  private static void isLinearRing(LineString lineString) {
```

Comment:
It never returned false. It would wither through an exception or return true, so the return value was useless anyway. Incidentally it is a private method and no internal caller would check the return value, so I removed it.

Ooops about the s. It will be gone very soon.